proposals/mips/mip-x39/mip-x39.sol

```diff
@@ -0,0 +1,174 @@
+//SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import "@forge-std/Test.sol";
+
+import {ChainlinkOracle} from "@protocol/oracles/ChainlinkOracle.sol";
+import {ChainlinkCompositeOracle} from "@protocol/oracles/ChainlinkCompositeOracle.sol";
+import {AggregatorV3Interface} from "@protocol/oracles/AggregatorV3Interface.sol";
+
+import {HybridProposal, ActionType} from "@proposals/proposalTypes/HybridProposal.sol";
+import {AllChainAddresses as Addresses} from "@proposals/Addresses.sol";
+import {MOONBEAM_FORK_ID, BASE_FORK_ID, OPTIMISM_FORK_ID} from "@utils/ChainIds.sol";
+import {ProposalActions} from "@proposals/utils/ProposalActions.sol";
+import {ChainIds} from "@utils/ChainIds.sol";
+
+/// @title MIP-X39: rETH Market Exchange Rate Feed and Debt Management
+/// @author Moonwell Contributors
+/// @notice Proposal to:
+///         1. Update rETH oracle on Base to use exchange-rate feed instead of market price
+///         2. Withdraw WETH reserves on Optimism and transfer to BAD_DEBT_REPAYER_EOA
```

Comment:
this value isn't used in the proposal script? do we need another action that sends from the temporal governor to `BAD_DEBT_REPAYER_EOA` ?